README.md

```diff
@@ -34,6 +34,7 @@ The following resources are currently supported:
 - `customer_groups`
 - `customers`
 - `orders`
+- `order_shipments`
```

Comment:
This should be removed now
```suggestion
```

Reply:
https://github.com/MedShift/bigc/pull/25/commits/bdfb8fb7cdc7d4338022155075cb983aa2a9af02

Reply:
https://github.com/MedShift/bigc/pull/25/commits/d0a052f9c6ae3579134c25f508c84f2ab7f4079f